data/mil2525d/utilities/style-utilities/merge-stylx-utilities/versions.csv

```diff
@@ -3,4 +3,4 @@ rule_engine_plugin_version,1.0.0
 pro_version,1.2.4375
 milstylx_version,1.2.0
 milstd_version,2525D-0
-jmsml_version,1.0.1
+jmsml_version,1.0.2
```

Comment:
@abouffard - are you sure this version is correct - since I know you were still making changes to make this process worked after this version/date - see commit history: https://github.com/Esri/joint-military-symbology-xml/commits/master since Nov 21